geometry_msgs/Point cartesianLocation
bool isVisible